coldfront/plugins/qumulo/fields.py

```diff
@@ -26,13 +26,10 @@ def to_python(self, value: list[str]):
 
 class StorageFileSystemPathField(forms.CharField):
     default_validators = [
+        validate_relative_path,
         validate_parent_directory,
         validate_filesystem_path_unique,
     ]
 
     def run_validators(self, value: str) -> None:
-        validate_relative_path(value)
-        storage_root = os.environ.get("STORAGE2_PATH").strip("/")
-        full_path = f"/{storage_root}/{value}"
-
-        return super().run_validators(full_path)
+        return super().run_validators(value)
```

Comment:
I believe you can remove this function entirely in this case.  It's just calling the super function and isn't doing anything anymore

Reply:
Good call

Reply:
Done.  Method removed.